posthog/models/person/person.py

```diff
@@ -188,13 +214,50 @@ class PendingPersonOverride(models.Model):
 
 
 class FlatPersonOverride(models.Model):
+    """
+    The (flat) person overrides model/table contains a consolidated record of
+    all merges that have occurred, but have not yet been integrated into the
+    ClickHouse events table through a squash operation. Once the effects of a
+    merge have been integrated into the events table, the associated override
+    record can be deleted from this table.
```

Comment:
If keeping these around is cheap it seems worthwhile to me. There's something nice about having a source of truth in a DB like PG.

But if I'm forgetting something about how squash works then ignore me, it's more of an idle thought.

Reply:
They are currently deleted after a squash happens, #19347 probably helps shine a lot more light on how that process works -- probably worth picking up this conversation over there? I could see the benefits of not deleting from this table, but would need to give it a closer look to check if that would be safe to do so.